lib/src/client/call.dart

```diff
@@ -50,14 +51,14 @@ const _reservedHeaders = [
 /// expected to modify the map before returning or before completing the
 /// returned [Future].
 typedef FutureOr<void> MetadataProvider(
-    Map<String, String> metadata, String uri);
+    Map<String, String?> metadata, String uri);
 
 /// Runtime options for an RPC.
 class CallOptions {
   final Map<String, String> metadata;
```

Comment:
Is this the same metadata that is `<String, String?>` elsewhere? Then, should likely be the same type.

Reply:
Changed to `<String, String>` everywhere